sentences/it/cover_HassCloseCover.yaml

```diff
@@ -12,10 +12,8 @@ intents:
         slots:
           device_class: "garage_door"
       - sentences:
-          - "<close> [la] (tenda|tapparella) [(<in> | <of>)] <area>"
-          - "<close> [le] (tende|tapparelle) [(<in> | <of>)] <area>"
-          - "<close> [<in>] <area> [la] (tenda|tapparella)"
-          - "<close> [<in>] <area> [le] (tende|tapparelle)"
+          - "<close> [(la | le)] (tenda | tende | tapparella | tapparelle) [(<in> | <of>)] <area>"
```

Comment:
```suggestion
          - "<close> [(la | le)] (tend(a | e) | tapparel(a | e) | venezian(a | e) ) [(<in> | <of>)] <area>"
```

Reply:
@alessandroias ok for me

Reply:
`veneziane` should be added in every file.
Also, we can use the singular/plural expansion in every file...

Reply:
> `veneziane` should be added in every file. Also, we can use the singular/plural expansion in every file...

I agree

Reply:
This sentence should be kept:
- "<close> [<in>] <area> [(la | le)] (tend(a | e) | tapparel(a | e) | venezian(a | e) )"